src/Commands/Extension/AzCommand.cs

```diff
@@ -72,43 +64,28 @@ protected override AzArguments BindArguments(ParseResult parseResult)
 
     private static string? FindAzCliPath()
     {
+        string executableName = "az";
```

Comment:
Is the correct on all plats, win, linux, mac.

Reply:
Yes, this is xplat and also part of our implementation of the identity library.